x/gov/spec/07_client.md

```diff
@@ -322,34 +348,66 @@ simd tx gov deposit 1 10000000stake --from cosmos1..
 
 #### submit-proposal
 
-The `submit-proposal` command allows users to submit a governance proposal and to optionally include an initial deposit.
+The `submit-proposal` command allows users to submit a governance proposal along with some messages and metadata.
+Messages, metadata and deposit are defined in a JSON file.
+
+```bash
+simd tx gov submit-proposal [path-to-proposal-json] [flags]
+```
+
+Example:
+
+```bash
+simd tx gov submit-proposal /path/to/proposal.json --from cosmos1..
+```
+
+where proposal.json contains:
+
+```bash
+{
+  "messages": [
+    {
+      "@type": "/cosmos.bank.v1beta1.MsgSend",
+      "from_address": "cosmos1...",
+      "to_address": "cosmos1...",
+      "amount":[{"denom": "stake","amount": "10"}]
+    }
+  ],
+  "metadata": "AQ==",
+  "deposit": "10stake"
+}
+```
+
+#### submit-legacy-proposal
+
+The `submit-legacy-proposal` command allows users to submit a governance legacy proposal along with an initial deposit.
 
 ```bash
-simd tx gov submit-proposal [command] [flags]
+simd tx gov submit-legacy-proposal [command] [flags]
 ```
 
 Example:
 
 ```bash
-simd tx gov submit-proposal --title="Test Proposal" --description="testing, testing, 1, 2, 3" --type="Text" --deposit="10000000stake" --from cosmos1..
+simd tx gov submit-legacy-proposal --title="Test Proposal" --description="testing" --type="Text" --deposit="100000000stake" --from cosmos1..
 ```
 
-Example (`cancel-software-upgrade`):
+Example (`legacy-cancel-software-upgrade`):
 
 ```bash
-simd tx gov submit-proposal cancel-software-upgrade --title="Test Proposal" --description="testing, testing, 1, 2, 3" --deposit="10000000stake" --from cosmos1..
+simd tx gov submit-legacy-proposal legacy-cancel-software-upgrade --title="Test Proposal" --description="testing" --type="Text" --deposit="100000000stake" --from cosmos1..
```

Comment:
why adding --type flag here? it doesn't seem like it's actually there

Reply:
oh! you're right there. It's a mistake